homeassistant/components/airzone/sensor.py

```diff
@@ -0,0 +1,64 @@
+"""Support for the Airzone sensors."""
+from __future__ import annotations
+
+from aioairzone.const import AZD_NAME, AZD_ZONES
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import AirzoneDevice
+from .const import DOMAIN, SENSOR_TYPES
+from .coordinator import AirzoneUpdateCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Add Airzone sensors from a config_entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    sensors = []
+    for zone_id, zone_data in coordinator.data[AZD_ZONES].items():
+        zone_name = zone_data[AZD_NAME]
+
+        for description in SENSOR_TYPES:
+            if description.key in zone_data:
+                sensors.append(
+                    AirzoneSensor(
+                        coordinator,
+                        description,
+                        zone_id,
+                        zone_name,
+                    )
+                )
+
+    async_add_entities(sensors, False)
+
+
+class AirzoneSensor(AirzoneDevice, SensorEntity):
+    """Define an Airzone sensor."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        description: SensorEntityDescription,
+        zone_id: str,
+        zone_name: str,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator, zone_id, zone_name)
+        self._attr_name = f"{zone_name} {description.name}"
+        self._attr_unique_id = f"{DOMAIN}_{zone_id}_{description.key}"
```

Comment:
```suggestion
        self._attr_unique_id = f"{zone_id}_{description.key}"
```

Unique ids are per integration, per platform so the DOMAIN is not needed

Reply:
Done in 9a96604fc814869ecb594340ace7a1325230b05b